include/Addon/License/Licenser.php

```diff
@@ -68,6 +69,7 @@ public function load() {
 
 		add_action( 'el_before_addon_list', array( $this, 'render_bundle_license_form' ) );
 		add_action( 'el_before_logs_list_table', array( $this, 'render_more_fields_addon_upsell_message' ) );
+		add_action( 'el_before_logs_list_table', array( 'Upseller', 'render_auto_delete_logs_addon_upsell_message' ) );
```

Comment:
This should happen inside the `Upseller`.

Also move the Upsell message for more fields add-on to `Upseller`.

Reply:
Fixed.  Moving More fields add-on upsell message is creating few problems. So more fields addon upsell message is not moved.